index.html

```diff
@@ -947,6 +947,12 @@ <h3>Processing Model</h3>
 <!-- Source: https://docs.google.com/document/d/1I7XGNJ57Qgjkg9pL11s7MK7zGEcwAgdNj1W5f7NKbW8/ -->
 <img src="timestamp-diagram.svg" alt="Resource Timing attributes"
 style='margin-top: 1em'></figure>
+<aside class="note">
```

Comment:
Maybe "The phases illustrated above are not necessarily consecutive, and user agents may spend time between them on processing, waiting for TCP connections to become available, waiting before sending requests to prevent contention or any other implementation detail. The unaccounted time spent by the user agent can be calculated as the difference between the consecutive timestamps. Its sum can be calculated as the difference between the `duration` value and `responseEnd - startTime`. "

@nicjansma @igrigorik - thoughts?

Reply:
Hmm, tbh I find this really cryptic and hard to grok. As in, you have to be well versed in the nuance here to understand what this is saying. If our concern is specifically "stalled" / "blocked", I'd focus and lead with that.

"The user agent may inject other processing steps that are not included in the above diagram. For example, due to limitations of HTTP/1.x TCP connection limits, the user agent may be blocked waiting for an available socket, and such delays are not explicitly exposed by this specification."

~ish?

Reply:
As per the fact that many developers collect performance metrics based on this diagram, personally I prefer @yoavweiss 's version since it presents a clear formula for calculating "stalled" time